lang/main-fr.json

```diff
@@ -746,6 +746,7 @@
         "privacy": "Confidentialité",
         "recentList": "Récent",
         "recentListDelete": "Supprimer",
+        "recentListDeleteAll": "Supprimer l'historique",
```

Comment:
@damencho what is the status of the translations? Do we update the other languages manually?

Reply:
I don't know how you do usually, I can add the line in each file manually...

Reply:
@damencho Will know I hope :) Please hold on this one!

Reply:
Just edit main.json if needed and do not touch the rest of the files, please. Thanks.

Reply:
Done ! :) I'll remember :)